autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -104,6 +103,7 @@ def __init__(self, path: str, problem_type: str, scheduler_options=None, objecti
         self.model_pred_times = {}
         self.models = {}
         self.model_graph = nx.DiGraph()
+        self.model_compressed_dict = {}  # Dict of Uncompressed Model -> Compressed Model
```

Comment:
we'll need to standardize terminology between "FULL" and "COMPRESSED".  I prefer "FULL" as it more clearly warns against interpreting accuracies on any of the provided data.

Reply:
Agreed, changed all references to compressed to full